### PR TITLE
enable patient experience in production

### PR DESCRIPTION
**Pull request by timothybest-usds**
## Related Issue or Background Info

enable patient experience in production

https://usds.slack.com/archives/C01LTSNKEPP/p1614946649154600?thread_ts=1614917629.154000&cid=C01LTSNKEPP

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
...
     "deploy-azure-stg": "REACT_APP_OKTA_URL=https://stg.simplereport.gov REACT_APP_BACKEND_URL=https://stg.simplereport.gov/api PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportstgapp --destination-path '/app'",
-    "deploy-azure-prod": "REACT_APP_APPINSIGHTS_KEY=ec65665e-358e-4f07-9292-3d80f23bcc3b REACT_APP_OKTA_URL=https://simplereport.gov REACT_APP_BACKEND_URL=https://simplereport.gov/api PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportprodapp --destination-path '/app'",
+    "deploy-azure-prod": "REACT_APP_APPINSIGHTS_KEY=ec65665e-358e-4f07-9292-3d80f23bcc3b REACT_APP_OKTA_URL=https://simplereport.gov REACT_APP_BACKEND_URL=https://simplereport.gov/api PUBLIC_URL=/app/ REACT_APP_PATIENT_EXPERIENCE_ENABLED=true yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportprodapp --destination-path '/app'",
```
Comment (nickscialli-usds) [inline frontend/package.json:77]: Can we just flip this on in the frontend `.env` file? That what we've been doing with the other flags I think.

Comment (timothybest-usds) [reply frontend/package.json]: updated to flip the flag on for every env.

**Review by nickscialli-usds — APPROVED** [review]
🚀

File: frontend/package.json (hunk 1 above)
Comment (benwarfield-usds) [reply frontend/package.json]: This seems like a situation that calls for an actual config file, but I have the feeling that configuration management for multiple deployed environments is a rabbit hole that yarn has stored some traps in.

**Pull request merged**